packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -101,7 +103,7 @@ module.exports = (resolve, rootDir, isEjecting) => {
               chalk.bold('setupTestFrameworkScriptFile') +
               ' in your package.json.\n\n' +
               'Remove it from Jest configuration, and put the initialization code in ' +
-              chalk.bold('src/setupTests.js') +
+              chalk.bold(`src/setupTests.${useTypeScript ? 'ts' : 'js'}`) +
```

Comment:
Let's leave this as-is since we don't suggest the mirad of other supported extensions 
```suggestion
              chalk.bold('src/setupTests.js') +
```

Reply:
Reverted.